commit b4c85d459c3a2c539a2d5885b12372e404bafc6b
Author: Christian Lamparter <chunkeey@googlemail.com>
Date:   Sun Sep 26 21:40:40 2010 +0200

    carl9170: angle brackets for wiki link
    
    "The convention seems to be angle brackets around
    URLS in Kconfig."
     -- Finn Thain (to update web addresses in the kernel)
    
    Signed-off-by: Christian Lamparter <chunkeey@googlemail.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

--- a/drivers/net/wireless/ath/carl9170/Kconfig
+++ b/drivers/net/wireless/ath/carl9170/Kconfig
@@ -10,7 +10,7 @@ config CARL9170
 	  but it needs a special firmware (carl9170-1.fw) to do that.
 
 	  The firmware can be downloaded from our wiki here:
-	  http://wireless.kernel.org/en/users/Drivers/carl9170
+	  <http://wireless.kernel.org/en/users/Drivers/carl9170>
 
 	  If you choose to build a module, it'll be called carl9170.
 
